apps/compiler/includes/functions.sh

```diff
@@ -208,9 +208,9 @@ function conf_layer() {
         if grep -qE "^$KEY" "$BASE" && ! grep -qE "^$KEY.*=.*$VAL" "$BASE"; then
           # Replace line
           # Prevent issues with shell quoting 
-          sed -i \
-            's,^'"$KEY"'.*,'"$KEY = $VAL$COMMENT"',g' \
-            "$BASE"
+         TEMP="$(mktemp)"
```

Comment:
nitpick, `TEMP` is missing a space in its indentation. I probably left a bad "suggestion" block in the last comment

```suggestion
          TEMP="$(mktemp)"
```